mirobo/ceil_cli.py

```diff
@@ -120,19 +120,20 @@ def off(dev: mirobo.Ceil):
 
 
 @cli.command()
-@click.argument('level', callback=validate_bright, required=True,)
+@click.argument('level', callback=validate_percentage, required=True,)
 @pass_dev
-def set_bright(dev: mirobo.Ceil, level):
+def set_brightness(dev: mirobo.Ceil, level):
     """Set brightness level."""
-    click.echo("Brightness: %s" % dev.set_bright(level))
+    click.echo("Brightness: %s" % dev.set_brightness(level))
 
 
 @cli.command()
-@click.argument('level', callback=validate_bright, required=True,)
+@click.argument('level', callback=validate_percentage, required=True,)
 @pass_dev
-def set_cct(dev: mirobo.Ceil, level):
+def set_color_temperature(dev: mirobo.Ceil, level):
     """Set CCT level."""
-    click.echo("CCT level: %s" % dev.set_cct(level))
+    click.echo("Correlated color temperatur level: %s" %
```

Comment:
temperature. Any ideas what "correlated" means and does it bring anything for the user what simply calling it "color temperature" would not?

Reply:
I removed "correlated". It won't help anybody and isn't correct because a percentage is delivered.